docs/visual-basic/getting-started/whats-new.md

```diff
@@ -180,7 +180,7 @@ For more information, see [Reference Return Values](../programming-guide/languag
 
 ## Visual Basic 14
 
-[Nameof](../../csharp/language-reference/keywords/nameof.md)
+[Nameof](../../csharp/language-reference/operators/nameof.md)
```

Comment:
It seems there is no Visual Basic reference article for the `nameof` operator.

Reply:
/cc @rpetrusha Do you have a better link for this?

Reply:
There are two choices for the link, both of which should be temporary:
- The mention of `nameof` in https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new#visual-basic-14.
- The design notes, at <https://github.com/dotnet/vblang/blob/189d2c15703b91a9241ce9ade46e6bc24b193cd9/meetings/2014/README.md>.
There's an outstanding issue to add the nameof operator for VB that's assigned to me for this sprint (though it was also assigned to me in the last sprint, and perhaps the one before that). I'll note in the issue that the what's new should be modified to link to the new nameof article when it's ready.

Reply:
@rpetrusha the first suggested link would be a self-reference; the design notes, I'm afraid, would be more confusing than the existing link to the C# `nameof` article (design notes mix C# and VB examples). Thus, I suggest to keep the existing text as it is until the dedicated VB article is created. Please let me know if you prefer the link to the design notes, I'll update the text then.